### PR TITLE
Release: Update docs

### PR DESCRIPTION
**Pull request by ghost**
- Update docs structure.
- Remove outdated docs.
- Update deployment process notes.
- Reduce log level for "Scheduled task" message
- Don't set `ADMINS` during Vagrant deploy.
- Fix the color of the font in the calendar drop down

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,35 +24,70 @@ lint: lint-py lint-js
...
+pushmessages:
+	# Upload the latest English PO file to Transifex
+	tx push -s
+
+pullmessages:
+	# Pull the latest translated PO files from Transifex
+	tx pull -af
+
+setup:
+	virtualenv -p `which python3.4` $(WORKON_HOME)/tracpro
```
Comment (daaray) [inline Makefile]: Should this by `python2.7`?

Comment (rebecca-caktus) [inline Makefile]: Yep! Good catch.

**Comment by daaray** [comment]
👍

**Comment by emullaney** [comment]
:unicorn: :rainbow:

**Pull request merged**